### PR TITLE
BUG: Scrollpositionen är inte individuell för varje tabb. #483

### PR DESCRIPTION
**Pull request by Abdelkad3r**
The bug has been fixed by using scrollPosition instead local state in redux, we bring scrollPosition value which we saved in Reducer then implemented it in logViewer by using updateScrollPosition function instead setScrollPosition function inside useEffect hook instead useState hook.

### CODE REVIEW

**Comment by gustavjorlov** [comment]
I can't really see any tests proving the new logic to work?

**Comment by Abdelkad3r** [comment]
Yep, i did not test, i'm testing right now ...

File: src/js/view/components/LogViewer.js [hunk 1]
```diff
@@ -56,6 +56,11 @@ const LogViewer = props => {
   const lastSeenLogSize = props.lastSeenLogSizes[props.source.path]
     ? props.lastSeenLogSizes[props.source.path]
     : 0;
+  
+  const scrollPositionFromReducer = props.scrollPositions[props.source.path]
```
Comment (MelindaSW) [inline src/js/view/components/LogViewer.js]: The scrollPositionFromReducer can probably be renamed to just scrollPosition. When we pairprogrammed the other day I just wanted to call it something different from the variable to distinguish it from the state variable we were replacing.

File: src/js/view/reducers/logViewerReducer.js [hunk 2]
```diff
@@ -118,6 +119,19 @@ export const logViewerReducer = (state = initialState, action) => {
       };
     }
 
+    //updateScrollPosition action passed by Reducer
```
Comment (MelindaSW) [inline src/js/view/reducers/logViewerReducer.js]: This comment is probably not needed here. Or what do you think? I find the action.type explaining what is going on in itself.

**Comment by MelindaSW** [comment]
I tried it with files that were running and files that were not, longer files and shorter files. The position kept its place in all of the cases for me.

**Comment by MelindaSW** [comment]
Once the additional tests, and eventual corrections are in place this should be ready to be merged. I think there's an issue with the linting as well. Do you have the right settings in your Prettier extension?

File: src/js/view/components/LogViewer.js (hunk 1 above)
Comment (Abdelkad3r) [reply src/js/view/components/LogViewer.js]: I renamed scrollPositionFromReducer in LogViewer to scrollPosition.

File: src/js/view/reducers/logViewerReducer.js (hunk 2 above)
Comment (Abdelkad3r) [reply src/js/view/reducers/logViewerReducer.js]: Yes right that's not needed, so i just removed it.

**Comment by Abdelkad3r** [comment]
@MelindaSW i installed Prettier, but i'm not sure about the settings.

**Pull request closed without merging**